inbox/automatic-trust-transfer.xml

```diff
@@ -0,0 +1,414 @@
+	</p>
+	<p>
+		When using &xep0384;, a public identity key is transmitted over a channel which is not protected against active attacks.
+		That key has to be authenticated by the receiving device over a channel which is protected against active attacks to maintain the confidentiality of sent OMEMO messages and ensuring the authenticity and integrity of received OMEMO messages.
+	</p>
+	<p>
+		When using OMEMO, each device has a different identity key.
+		That makes it possible for new devices to use end-to-end encryption protecting against passive attacks without transmitting the private key over a secure channel from an existing device to the new one.
+		However, the downside of this approach is that it increases the number of authentications.
+		Without ATT all authentications have to be done manually.
+		With ATT though, only one mutal manual authentication per new key is required.
+	</p>
+</section1>
+<section1 topic='Glossary' anchor='glossary'>
+	<dl>
+		<di>
+			<dt>OMEMO message</dt>
+			<dd>Message encrypted with OMEMO</dd>
+		</di>
+		<di>
+			<dt>Device</dt>
+			<dd>See <cite>Device</cite> in &omemo-glossary;</dd>
+		</di>
+		<di>
+			<dt>Key</dt>
+			<dd>Public part of <cite>IdentityKey</cite> in &omemo-glossary;</dd>
+		</di>
+		<di>
+			<dt>Key identifier</dt>
+			<dd>Identifier for a key (e.g., a fingerprint or the key itself)</dd>
+		</di>
+		<di>
+			<dt>Key authentication</dt>
+			<dd>Verifying that a key received over an insecure channel is actually the one of the assumed device</dd>
+		</di>
+		<di>
+			<dt>Manual key authentication</dt>
+			<dd>Key authentication with user interaction (e.g., QR code scanning, fingerprint verification)</dd>
+		</di>
+		<di>
+			<dt>Automatic key authentication</dt>
+			<dd>Key authentication without user interaction (e.g., via ATT)</dd>
+		</di>
+		<di>
+			<dt>Mutual key authentication</dt>
+			<dd>Key authentication in which two devices authenticate the key of each other</dd>
+		</di>
+		<di>
+			<dt>Key revocation</dt>
+			<dd>Revoking the trust in a key</dd>
+		</di>
+		<di>
+			<dt>Manual key revocation</dt>
+			<dd>Key revocation with user interaction (e.g., clicking a "Revoke" button)</dd>
+		</di>
+		<di>
+			<dt>Automatic key revocation</dt>
+			<dd>Key revocation without user interaction (e.g., via ATT)</dd>
+		</di>
+		<di>
+			<dt>Trust message</dt>
+			<dd>
+				OMEMO message which indicates that specific keys can be trusted (authentication) or no longer trusted (revocation).
+				A trust message for a device's key sent to another device is a trust message that contains the key identifer of the given key for authentication or revocation.
+				If a trust message only contains key identifiers for authentication, it is called <cite>authentication message</cite>.
+				If it only contains key identifiers for revocation, it is called <cite>revocation message</cite>.
+			</dd>
+		</di>
+	</dl>
+</section1>
+<section1 topic='Advantages' anchor='advantages'>
```

Comment:
@jcbrand This section should answer your question more precisely.
It does not differentiate between own devices and devices of a contact, since that is not necessary.